documentation/administration/original-mets.md

```
@@ -0,0 +1,36 @@
+
+# **Original-METS**
+
+## **Functionality**
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)